includes/discovery/sensors/entity-sensor.inc.php

```diff
@@ -4,10 +4,10 @@
 if (empty($entity_array)) {
     $entity_array = array();
     echo ' entPhysicalDescr';
-    $entity_array = snmpwalk_cache_multi_oid($device, 'entPhysicalDescr', $entity_array, 'CISCO-ENTITY-SENSOR-MIB');
+    $entity_array = snmpwalk_cache_multi_oid($device, 'entPhysicalDescr', $entity_array, 'ENTITY-MIB:CISCO-ENTITY-SENSOR-MIB');
```

Comment:
CISCO-ENTITY-SENSOR-MIB was already there, I didnt removed just for not break any other install and I dont have any other device using ENTITY-MIB for sensors so cant test if I remove it will break anything. Do you want me to remove anyways?